inst/templates/study_schema.R

```diff
@@ -0,0 +1,28 @@
+#' Study Schema for {{ study_name }}
+#'
+#' To source the study schema in a chunk in your Rmarkdown report:
+#' source("R/study_schema.R")
+#'
+#' @return a kable table with the study schema
+#' @export
+#'
+#' @examples
+study_schema <- function() {
+
+  caption <- "{{ study_name }} study schema."
```

Comment:
can this be a param too? If so then this could be a standalone function as well.

```
caption = "{{ study_name }} study schema."
```